### DISCUSSION TITLE
Server Errors/incompatible

### DISCUSSION THREAD

**Discussion by Astargos** [TrinityCore/TrinityCore#28230] [Q&A] [answered]
Hello, im trying to make a shadowlands servers for testing, but when i start the server i get a lot of errors, then the realm says its incompatible and kicks me out
(i have followed the installation guide and my client is the lastest from blizzard)

[Server.log](https://github.com/TrinityCore/TrinityCore/files/9483501/Server.log)

**Comment by Aokromes** [comment, marked as answer]
retail downgraded version.
